pallets/ah-migrator/src/lib.rs

```diff
@@ -905,4 +954,17 @@ pub mod pallet {
 			AhMigrationStage::<T>::get().is_finished()
 		}
 	}
+
+	// To be used for `IsTeleport` filter. Disallows DOT teleports during the migration.
```

Comment:
I do not know the impact from disabling the teleports during migration, but I think it is possible with low cost to keep them working. 
We could just have one migration stage that would sync up the checking account. 
For account migration I am actually checking out the migrated balances and had a plan to filter out the checking account, and not teleport it.